src/test/suite/fixtures/recorded/surroundingPair/takeFirstChar5.yml

```diff
@@ -0,0 +1,24 @@
+spokenForm: take first char
+languageId: typescript
+command:
+  actionName: setSelection
+  partialTargets:
+    - type: primitive
+      selectionType: token
+      modifier: {type: subpiece, pieceType: character, anchor: 0, active: 0}
+  extraArgs: []
+marks: {}
+initialState:
+  documentContents: //**
+  selections:
+    - anchor: {line: 0, character: 2}
+      active: {line: 0, character: 2}
+finalState:
+  documentContents: //**
+  selections:
+    - anchor: {line: 0, character: 0}
```

Comment:
I might have expected it to pick `**`, tho this one is def a corner case

Reply:
If all equal it chooses the one to the left. Of course we could continue to tweak these corner cases but I think this should work for all practical cases.

Reply:
Yeah kind of a toss up.  I'd assume it picks token it's at the start of, but I don't think there's much of a convention here.  This decision is probably fine